src/net/sourceforge/kolmafia/request/PingRequest.java

```diff
@@ -12,18 +12,15 @@ public class PingRequest extends GenericRequest {
   // api.php's responseText is about 1/4 the size of main.php's - and
   // measured ping time is about 1/4 as long.
 
-  public PingRequest() {
```

Comment:
Why did you delete this constructor?

Reply:
This constructor isn't used as far as I can tell.